docs/src/templates/pages/styleguide/ui-components/alerts.hbs

```diff
@@ -0,0 +1,134 @@
+---
```

Comment:
This looks like an exact copy of [the badges page](https://github.com/justeat/global-component-library/blob/master/docs/src/templates/pages/styleguide/ui-components/badges.hbs)?

Reply:
Yep – I wasn't meant to commit this… 🤦‍♂️

Reply:
Have removed this now – good catch ⚾️